src/components/ArpSpoof/ArpSpoof.tsx

```diff
@@ -1,8 +1,8 @@
-import { Button, Stack, TextInput, Alert } from "@mantine/core";
+import { Button, Stack, TextInput } from "@mantine/core";
 import { useForm } from "@mantine/form";
 import { useState, useEffect, useCallback } from "react";
 import { CommandHelper } from "../../utils/CommandHelper";
-import { UserGuide } from "../UserGuide/UserGuide";
+import { RenderComponent, UserGuide } from "../UserGuide/UserGuide";
```

Comment:
Remove UserGuide. Not required. 
```suggestion
import { RenderComponent } from "../UserGuide/UserGuide";
```